### PR TITLE
Fix the position of 'Forgot password?' on the remove device dialog

### PR DESCRIPTION
**Pull request by makingthematrix**
#### APK
[Download build #10848](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10848/artifact/build/artifact/wire-dev-PR1562-10848.apk)
[Download build #10924](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10924/artifact/build/artifact/wire-dev-PR1562-10924.apk)
[Download build #11026](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11026/artifact/build/artifact/wire-dev-PR1562-11026.apk)

### CODE REVIEW

**Comment by someoneEsle** [comment]
- [ ] Bug 1
Text is not in the right font and is not aligned to the input field above (design comments)

File: app/src/main/res/layout/remove_otr_device_dialog.xml [hunk 1]
```diff
@@ -18,28 +18,50 @@
...
+    </ScrollView>
+
+    <com.waz.zclient.ui.text.TypefaceTextView
+        android:id="@+id/device_forgot_password"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/new_reg__password__forgot"
+        android:textAllCaps="true"
+        android:layout_marginStart="@dimen/wire__padding__4"
+        android:textColor="@color/accent_blue"
```
Comment (deanrobertcook) [inline app/src/main/res/layout/remove_otr_device_dialog.xml]: not sure if this is the right colour? But, if it looks okay, then sure

**Comment by someoneEsle** [comment]
build again

**Comment by someoneEsle** [comment]
build again

**Pull request merged**